v2.0/app/utils/messenger.js

```diff
@@ -21,40 +21,42 @@ export async function apiCall(
   const data = {
     __is_prod: IS_PROD || IS_CANARY,
     __is_admin_site: true,
-    ...dataToSend
+    ...dataToSend,
   };
 
   const formData = new FormData();
-  Object.keys(data).map(k => (formData.append(k, data[k])));
+  Object.keys(data).map((k) => formData.append(k, data[k]));
 
   if (!destinationUrl || destinationUrl.length < 2) {
-    return { success: false, error: 'Invalid URL passed to apiCall' };
+    return { success: false, error: "Invalid URL passed to apiCall" };
   }
 
   // make leading '/' optional
-  if (destinationUrl.charAt(0) === '/') {
+  if (destinationUrl.charAt(0) === "/") {
     destinationUrl = destinationUrl.substring(1);
   }
 
-  if (IS_LOCAL) {
-    destinationUrl = "api/" + destinationUrl;
-  }
-  
+  // if (IS_LOCAL) {
```

Comment:
This will break working locally.

Reply:
No, not with Sam's new changes. Infact, admin-frontend does not work when that part is not commented out. None of the links to the backend work.

Reply:
That change was needed because of Sam's changes, and he approved this.  So I don't quite understand.